sdk/python/kfp/cli/pipeline.py

```diff
@@ -61,9 +61,15 @@ def upload(ctx, pipeline_name, package_file):
     help="Name of the pipeline version",
     required=True
 )
+@click.option(
+    "-u",
+    "--update-default-version",
+    help="Update the default version",
+    required=False
+)
```

Comment:
Can we make this a boolean type?

Reply:
Yes, make sens updated it.

Reply:
There is no typing for the other parameters, so suggest we leave it. Update in `sdk/python/kfp/_client.py`